thinc/__init__.py

```diff
@@ -3,3 +3,9 @@
 
 from .about import __version__
 from .config import registry
+
+__all__ = [
+    "registry",
+    "__version__",
+    "numpy"
```

Comment:
I don't think we should reexport `numpy` anywhere.

Reply:
I agree. I already have one exception for `thinc/__init__.py`, which is to consider `__version__` as to be exported despite being prefixed with `_` - I could another one? Or we don't treat `thinc/__init__.py` as one of the modules that reexport everything they import.

Reply:
Maybe try to figure out whether the import is actually necessary now?

Reply:
Internal discussion indicates that this is likely still necessary. I added an exception for `numpy`, but I'm open to alternative suggestions.